profile/README.md

```diff
@@ -1,2 +1,13 @@
-The Concepticon project presents an attempt to link the large amount of different concept lists which are used in the linguistic literature, 
+The Concepticon project is an attempt to link the large amount of different concept lists which are used in the linguistic literature, 
 ranging from Swadesh lists in historical linguistics to naming tests in clinical studies and psycholinguistics.
+
+The resulting Concepticon (in the sense of [Poornima and Good 2010](https://aclanthology.org/W10-2101.pdf)) facilitates
+- comparison of lexical data across languages and collection contexts and
+- cross-linguistic comparison of data associated with word forms or concepts collected in the NoRaRe sub-project.
+
+The curation and publication of the Concepticon and NoRaRe data are organized as follows:
+
+1. "Raw" data is curated in the repositories [concepticon-data](https://github.com/concepticon/concepticon-data) and [norare-data](https://github.com/concepticon/norare-data), respectively.
+2. Re-usable, aggregated data is provided in [CLDF format](https://cldf.clld.org), via releases of the repositories [concepticon-cldf](https://github.com/concepticon/concepticon-cldf/releases) and [norare-cldf](https://github.com/concepticon/norare-cldf/releases), uploaded to Zenodo.
+3. The latest release of both projects is also served - for interactive browsing - by [clld apps](https://github.com/clld/clld) - at https://concepticon.clld.org and https://norare.clld.org respectively - 
```

Comment:
Maybe remove the final `-` after respectively and start a new sentence?

E.g. `and https://norare.clld.org, respectively. The source code for the Concepticon CLLD app can be found here and here for NoRaRe.` where here links to the respective repositories.